mne_bids/utils.py

```diff
@@ -814,14 +952,14 @@ def _gen_bids_basename(*, subject=None, session=None, task=None,
                f'but received: {on_invalid_er_task}')
         raise ValueError(msg)
 
-    order = OrderedDict([('sub', subject),
-                         ('ses', session),
+    order = OrderedDict([('sub', sub),
```

Comment:
why do you change this throughout the codebase?

Reply:
Was playing around w/ refactoring and think this got changed by accident. Sorry!